wps-cloudformation-template.yaml

```diff
@@ -228,16 +213,16 @@ Resources:
         Environment:
           -
             Name: "OUTPUT_S3_FILENAME"
-            Value: !Ref s3OutputFilename
+            Value: !FindInMap [ConfigurationFileMap, "Filename", "output"]
           -
             Name: "OUTPUT_S3_BUCKET"
-            Value: !Ref s3ConfigBucket
+            Value: !If [CreateEphemeralBucket, !Ref EphemeralS3Bucket, !FindInMap [EnvironmentMap, !Ref environment, BucketName]]
           -
             Name: "STATUS_S3_BUCKET"
-            Value: !Ref s3ConfigBucket
+            Value: !If [CreateEphemeralBucket, !Ref EphemeralS3Bucket, !FindInMap [EnvironmentMap, !Ref environment, BucketName]]
           -
             Name: "STATUS_S3_FILENAME"
-            Value: !Ref s3StatusFilename
+            Value: !FindInMap [ConfigurationFileMap, "Filename", "status"]
```

Comment:
We will improve the code to refer to only one bucket. As part of this task, I have just automated the manual step of putting the config files up on s3.